components/post_view/reaction_list/reaction_list.jsx

```diff
@@ -123,7 +123,7 @@ export default class ReactionList extends React.PureComponent {
                 <Tooltip id='addReactionTooltip'>
                     <FormattedMessage
                         id='reaction_list.addReactionTooltip'
-                        defaultMessage='Add reaction'
+                        defaultMessage='Add a reaction'
```

Comment:
Question: The majority of items have different casing to this (e.g., Written like This). Is this different based on context or a sub-level of content?

Reply:
Same question applies to a few other entries below.

Reply:
These are tooltips, hence sentence case per the UI guidelines